typesafety/autodecorator.py

```diff
@@ -59,26 +63,39 @@ def __is_attribute_mutable(self, object_dict, attr):
 
         return attr in object_dict['__slots__']
 
-    def __decorate(self, module, *, use_dict):
+    def __decorate_module(self, module, *, use_dict):
+        for key, value in self.__iterate_decorables(use_dict):
+            if self.__is_object_native(module, value):
+                continue
+
+            self.__decorate_item(module, key, value)
+
+    def __is_object_native(self, module, obj):
```

Comment:
If I understand correctly, this helper method answers the question `__is_external`, but its name seems to suggest the opposite.